src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1393,8 +1381,7 @@ public Territory getRocketAttack(final Collection<Territory> candidates, final T
   }
 
   public static int save(final String filename, final GameData m_data) {
-    try (FileOutputStream fos = new FileOutputStream(filename);
-        ObjectOutputStream oos = new ObjectOutputStream(fos);) {
+    try (FileOutputStream fos = new FileOutputStream(filename); ObjectOutputStream oos = new ObjectOutputStream(fos);) {
```

Comment:
okay. I also cut a ticket for the larger problem: https://github.com/triplea-game/triplea/issues/1507, with a quick grep it was not too hard to find all such cases.